test_requests.py

```diff
@@ -1261,6 +1262,20 @@ def test_preserve_last_key_case(self):
         assert frozenset(cid.keys()) == keyset
         assert frozenset(cid) == keyset
 
+    def test_copy(self):
+        cid = CaseInsensitiveDict({
+            'Accept': 'application/json',
+            'user-Agent': 'requests',
+        })
+        assert cid == cid.copy()
+
+    def test_repr(self):
```

Comment:
I only ever test reprs when I think they're an important part of the API. In this case I don't really mind.